src/reducers/tests/reducers.test.js

```diff
@@ -0,0 +1,42 @@
+import expect from 'expect';
+import combinedReducers from '../index';
+import userReducer from '../userReducer';
+import articleReducer from '../articleReducer';
+import ACTION_TYPES from '../../actions/actionTypes';
+import initialState from '../../commons/initialStates';
+
+describe('post reducer', () => {
+  it('should returns the initial state', () => {
+    expect(combinedReducers(undefined, {})).toEqual(initialState);
```

Comment:
why are you passing undefined into this test?

Reply:
I am passing the state as undefined so that it can bring back the initial state.